src/pages/detailedpokemon/[id].tsx

```diff
@@ -225,8 +280,128 @@ export default function DetailedPokemon() {
                             ></audio>
                         </button>
                     </div>
+                    {/* evolution wrapper  */}
+                    <div className="grid grid-cols-3 gap-x-2 gap-y-1 col-span-full border-2 border-black rounded-lg p-2 bg-white/20">
+                        {evolutionSpecies.length > 0 &&
+                            evolutionNames.length > 0 && (
+                                <Link
+                                    href={`/detailedpokemon/${evolutionSpecies[0]?.id}`}
+                                    className="text-xl border-2 border-black rounded-lg p-1 gap-0.5  hover:bg-white/30 transition-colors duration-300"
+                                >
+                                    {evolutionNames[0]?.name}
+                                </Link>
+                            )}
+                        {evolutionChain.chain.evolves_to.map((chainLink) => {
+                            return (
+                                <EvolvesTo
+                                    key={chainLink.species.name}
+                                    chainLink={chainLink}
+                                    evoStage={1}
+                                    evolutionSpecies={evolutionSpecies}
+                                />
+                            );
+                        })}
+                    </div>
                 </div>
             </div>
         </Layout>
     );
 }
+
+type EvolvesToProps = {
+    chainLink: ChainLink;
+    evoStage: number;
+    evolutionSpecies: PokemonSpecies[];
+};
+
+function EvolvesTo({ chainLink, evoStage, evolutionSpecies }: EvolvesToProps) {
+    let className;
+    if (evoStage < 1) {
+        console.error("Value of evoStage is <1");
+        return null;
+    } else if (evoStage === 1) {
```

Comment:
Instead of defining `let className` here, please define it inline in the element where you are applying `className`.

The conditions (`evoStage === 1`, etc.) and their resulting values (`col-start-1`, etc.) can be interpolated into that string.